### DISCUSSION TITLE
AFrame Object Material Color

### DISCUSSION THREAD

**Discussion by georgebutler** [mozilla/hubs#4105] [Hubs Cloud Questions]
Hey All,

Currently spawning a custom A-Frame object with the flat shader and yellow color. We're having some issues with changing the attribute color at runtime, as it seems to be stuck on the yellow color. Would it be better for us to create different instances of the object with the various possible colors or is there a way to set the color at runtime? _Any help is appreciated!_

**Comment by pattersonbl2** [comment]
Are you still having issues with this ?
